flambe/compile/serialization.py

```diff
@@ -49,7 +49,9 @@ class State(OrderedDict):
 # Private Helpers
 
 def _convert_to_tree(metadata: Dict[str, Any]) -> SaveTreeNode:
-    tree = SaveTreeNode(state={}, version=metadata[''][VERSION_KEY],
+    new_state = OrderedDict()
+    new_state._metadata = {}
```

Comment:
nit

Is it worth creating a different class for this instead of adding an attribute on the `OrderedDict`?

Reply:
This is done to be compatible with torch